doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -4497,3 +4497,26 @@ A curl example using allowing access to a dataset's metadata
 
 Please see :ref:`dataverse.api.signature-secret` for the configuration option to add a shared secret, enabling extra
 security.
+
+Send Feedback To Contact(s)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
We should probabl add something like this:

See also Return a Dataset to Author

It's very similar: https://guides.dataverse.org/en/5.13/api/native-api.html#return-a-dataset-to-author

And we should probably add a see also from there back to here.